src/current/v23.1/essential-metrics.md

```diff
@@ -0,0 +1,135 @@
+| sql.txn.begin.count                                 | sql.txn.begin.count                                          | SQL                     | Number of SQL transaction BEGIN statements successfully executed | This metric reflects workload volume by counting explicit [transactions](transactions.html). Use this metric to determine whether explicit transactions can be refactored as implicit transactions (individual statements). |
+| sql.txn.commit.count                                | sql.txn.commit.count                                         | SQL                     | Number of SQL transaction COMMIT statements successfully executed | This metric shows the number of [transactions](transactions.html) that completed successfully. This metric can be used as a proxy to measure the number of successful explicit transactions. |
+| sql.txn.rollback.count                              | sql.txn.rollback.count                                       | SQL                     | Number of SQL transaction ROLLBACK statements successfully executed | This metric shows the number of orderly transaction [rollbacks](rollback-transaction.html). A persistently high number of rollbacks may negatively impact the workload performance and needs to be investigated. |
+| sql.txn.abort.count                                 | sql.txn.abort.count                                          | SQL                     | Number of SQL transaction abort errors                       | This high-level metric reflects workload performance. A persistently high number of SQL transaction abort errors may negatively impact the workload performance and needs to be investigated. |
+| sql.service.latency-p90, sql.service.latency-p99    | sql.service.latency                                          | SQL                     | Latency of SQL request execution                             | These high-level metrics reflect workload performance. Monitor these metrics to understand latency over time. If abnormal patterns emerge, apply the metric's time range to the [**SQL Activity** pages](monitoring-and-alerting.html#sql-activity-pages) to investigate interesting outliers or patterns. In 23.1, the [**Statements page**](ui-statements-page.html) has P90 Latency and P99 latency columns to enable correlation with this metric. |
```

Comment:
Remove 23.1? 

I'm assuming this doc will only be published for 23.1 correct?

Reply:
Removed 23.1